docs/node-operators/staking-service-guidelines.mdx

```diff
@@ -109,6 +109,8 @@ Since each VRF is compared against this threshold (which is between 0 and 1), an
 
 As an example, let's say there is an account on the network which has $10^6$ (1 million) mina tokens in the staking ledger epoch $ep$. This same staking ledger for epoch $ep$ has a total supply of $8\times10^8$ (800 million) mina tokens. We can compute the odds that this account will win an individual slot in $ep$ using $\phi\left(\dfrac{10^6}{8\times10^8}\right) = 0.0017313674$. This gives us a $\sim0.17\%$ chance that this account will win each slot during epoch $ep$. We can then compute the mean number of blocks we expect this account to produce for this epoch by multiplying the result by $7140$, giving us a probabilistic mean of $\sim12.36$ blocks for the epoch.
 
+Another way to calculate the probability is $`1 - 0.25^{\left(\dfrac{10^6}{8\times10^8}\right)}`$ which gives the same result as above.
```

Comment:
I think you can remove the two back ticks, they are being rendered - see https://docs2-git-fork-stevenpack-patch-1-minadocs.vercel.app/node-operators/staking-service-guidelines#sending-many-transactions